src/index.html

```diff
@@ -11,7 +11,134 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="container">
+        <legend>Personal information</legend>
+        <label class="form_field">
+        Surname:
+          <input
+            type="text"
+            name="surname" 
+            autocomplete="off"
+          >
+        </label>
+        <label class="form_field">
+        Name:
+          <input
+            type="text" 
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form_field">
+        How old are You?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="form_field">
+        Full date of birth:
+          <input type="date" name="birthday">
+        </label>
+        <label class="form_field">
+        I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="terms"
+            required 
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Registration</legend>
+        <label class="form_field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form_field">
+        Password:
+          <input
+            type="password" 
+            name="password" 
+            minlength="5"
+            maxlength="20"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+        <label class="form_field">
+        Do you love cats?
+        <input
+          type="radio"
+          name="cats"
+          value="Yes"
+        >
+        Yes
+          <input 
+            type="radio"
+            name="cats" 
+            value="No"
+          >
+        No 
+        </label>
```

Comment:
each input should have label
when I click on `No` text, I expecting `no` input radio to be selected